doc/api/cli.md

```diff
@@ -412,6 +412,16 @@ added: v9.0.0
 Specify the `module` of a custom [experimental ECMAScript Module loader][].
 `module` may be either a path to a file, or an ECMAScript Module name.
 
+### `--insecure-http`
+<!-- YAML
+added: FIXME
+-->
+
+Use an insecure HTTP parser that accepts invalid HTTP headers. This may allow
+interoperability with non-conformant HTTP implementations. It may also allow
+request smuggling and other HTTP attacks that rely on invalid headers being
+accepted.
```

Comment:
Should we say that it is highly unrecommended to use it?

Reply:
Ok, added to the "stove is hot enough to burn you" statement a "do not touch stove" statement :-)

> Avoid using this option.

@trott didn't you just rework all the not recommended? Is the above the new form?

Reply:
> @Trott didn't you just rework all the not recommended? Is the above the new form?

There's still a lot of "it is not recommended to use thing X" in the docs, but yes, I've been removing it when I come across it and replacing it with either:

* "Avoid X."

or: 

* "Do not use X."

In this context, I guess "avoid" is better than "do not use", but whichever is fine by me.